cylc/flow/data_messages.proto

```diff
@@ -230,6 +230,7 @@ message PbTaskProxy {
     optional bool flow_wait = 27;
     optional PbRuntime runtime = 28;
     optional int32 graph_depth = 29;
+    optional string run_mode = 30;
```

Comment:
Should be in `PbRuntime` instead.

Reply:
After broadcasts I'm pretty sure

Reply:
> Should be in `PbRuntime` instead.

Why? This represents the `itask.run_mode` property.

Reply:
This should match the schema